src/gantry/widgets.py

```
@@ -347,3 +348,177 @@ def _build_text(self) -> str:
+        items = self._list_view_items[lv_id]
+        if 0 <= idx < len(items):
+            name, implemented = items[idx]
+            self.post_message(self.ResourceSelected(name, implemented))
+
+    def focus_first_item(self) -> None:
+        """Focus the first inner ListView (Workloads group)."""
+        try:
+            first_lv_id = next(iter(self._list_view_items))
+            self.query_one(f"#{first_lv_id}", ListView).focus()
+        except (StopIteration, Exception):
+            pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)